### ISSUE TITLE
Implement org chart graph

### ISSUE THREAD

**Issue by mbland** [18F/team_api#19]
This is what was https://team-api.18f.gov/api/departments/ was intended to become, but just haven't gotten around to yet. (Also, per @ultrasaurus's suggestion in that issue, we should pick a different name than "departments" since that's an overloaded term in government.)

Replaces 18F/hub#446.

cc: @kaitlin @harrisj

**Comment by ertzeid** [comment]
Is this intended to show who's on what team? Or more of an overall view of team (eng, ux, delivery, etc.) structure? Happy to pick this up, but not entirely sure what we want it to look like.

**Comment by mbland** [comment]
@ertzeid Yes. :smiley: Let's discuss this more in the office tomorrow, and then update this issue with next steps for vetting our ideas with taxonomy-minded folks.
